faithcomesbyhearing/dbp-web
# FCBHDBP-579 bibleis-web: invalid CDN request

# Description
The issue causing an infinite loop when a video chunk failed has been resolved. The fix involved eliminating a poor practice of retrying a failed request. Additionally, I've mitigated around 70 vulnerabilities by overriding three packages and removing some that are no longer necessary. Finally, I have overwritten the Load Policies: `playlistLoadPolicy`, `fragLoadPolicy` and `keyLoadPolicy` to use the exponential strategy for the backoff parameter.

However, I am still encountering some CORS errors for specific videos, such as /bible/FRAPDF/MAT/1.

This PR is related with the API DBP PR:
- https://github.com/faithcomesbyhearing/dbp/pull/932

## Related Issue

https://fullstacklabs.atlassian.net/browse/FCBHDBP-580

## How to Test
- You should go the path: `bible/EN1ESV/MAT/1` and it must load the correct page and the console doesn't render errors. Also, the CDN URL must be: `d1gd73roq7kqw6.cloudfront.net`
- You should go the path: `bible/FRAPDF/MAT/1` and it must load the correct page and the console doesn't render errors. Also, the CDN URL must be: `d1gd73roq7kqw6.cloudfront.net`
